optional-requirements.txt

```diff
@@ -0,0 +1,8 @@
+# scipy
+scipy
```

Comment:
This is only kept for BC, because previously `pip install torchvision[scipy]` would also install `scipy`. Happy to remove it in case we don't want to keep BC here.